sql/src/main/java/io/crate/metadata/RelationInfo.java

```diff
@@ -60,6 +62,10 @@ public String pretty() {
 
     List<ColumnIdent> primaryKey();
 
+    default List<AnalyzedCheckConstraint> checkConstraints() {
+        return ImmutableList.of();
```

Comment:
```suggestion
        return List.of();
```

Would avoid using guava if not necessary